partner_industry_secondary/readme/CONTRIBUTORS.rst

```diff
@@ -7,3 +7,4 @@
 * Jordi Ballester Alomar <jordi.ballester@eficent.com>
 * Miquel Raïch <miquel.raich@eficent.com>
 * Cristina Martin R.
+* Eduardo de Miguel <edu@moduon.team> (`Moduon <https://www.moduon.team/>`__)
```

Comment:
suggestion: don't put your email if you don't want people to contact you directly when they have issues.